Makefile

```diff
@@ -15,8 +15,14 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
+# Docker build config variables
+CREDENTIALS_PATH ?= /cft/workdir/credentials.json
+DOCKER_ORG := gcr.io/cloud-foundation-cicd
+DOCKER_TAG_BASE_KITCHEN_TERRAFORM ?= 2.3.0
+DOCKER_REPO_BASE_KITCHEN_TERRAFORM := ${DOCKER_ORG}/cft/kitchen-terraform:${DOCKER_TAG_BASE_KITCHEN_TERRAFORM}
+
 # All is the first target in the file so it will get picked up when you just run 'make' on its own
-all: check_shell check_python check_golang check_terraform check_docker check_base_files check_trailing_whitespace generate_docs
+all: check_shell check_python check_golang check_terraform check_base_files check_trailing_whitespace generate_docs
```

Comment:
Target `all` seem to be altered to include only `doc generator` in the latest versions of [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/Makefile#L29-L33) and [module template](https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/Makefile#L29-L33) instead the checks migrated into `check` target.

Do we want to log a separate issue to back-port that change to all the modules or we do this in scope of the migration? 

cc: @aaron-lane @nick4fake @ingwarr @paulpalamarchuk

Reply:
This is made obsolete by the new development image and build process.